crates/cargo-test-macro/Cargo.toml

```diff
@@ -1,13 +1,20 @@
 [package]
 name = "cargo-test-macro"
-version = "0.1.0"
+version = "0.1.1"
 edition.workspace = true
+rust-version = "1.76.0"  # MSRV:1
 license.workspace = true
 homepage.workspace = true
 repository.workspace = true
-documentation = "https://github.com/rust-lang/cargo"
+documentation = "https://docs.rs/cargo-test-macro"
 description = "Helper proc-macro for Cargo's testsuite."
-publish = false
+include = [
```

Comment:
Guess we can just publish everything?

Reply:
I simply made these crates have a manifest similar to the other ones, in this aspect also. I'll remove the `include` if you want though.

Reply:
`home` is the only one there because it was imported from an external crate. Lean towards removing :)